src/status_im/chat/screen.cljs

```diff
@@ -44,7 +45,7 @@
       [toolbar/actions [{:icon      :icons/options
                          :icon-opts {:color :black}
                          :handler   #(on-options chat-id name group-chat public?)}]]]
-     [add-contact-bar]]))
+     (when-not public? [add-contact-bar])]))
```

Comment:
as i understand we want to show only for 1-1 chat? so we should check  `group-chat` too

Reply:
correct, thanks